lib/TPP/ToBlockLayoutAndBack.cpp

```diff
@@ -383,47 +385,108 @@ mlir::linalgx::packMatmulOp(RewriterBase &rewriter, linalg::MatmulOp matmulOp,
   return replacementOp;
 }
 
-FailureOr<vnni::MatmulOp>
+bool isVNNIPacked(linalg::GenericOp matmulOp) {
+  // TODO add VNNI packing checks here
```

Comment:
should you check the body too? `tpp::utils::hasMatmulBody`?

Reply:
I check that where I call isVNNIPacked method.